django_rest_multitokenauth/views.py

```diff
@@ -66,6 +64,10 @@ def post(self, request, *args, **kwargs):
 
         user = serializer.validated_data['user']
 
+        superuser_login_enabled = getattr(settings, 'AUTH_ENABLE_SUPERUSER_LOGIN', True)
```

Comment:
This should default to `False` so it does not affect existing applications that upgrade to a newer release.